Library/Homebrew/diagnostic.rb

```diff
@@ -1069,6 +1069,19 @@ def check_for_external_cmd_name_conflict
         message
       end
 
+      def check_for_member_of_admin_group
+        groups = Utils.popen_read("groups").split
+        return unless groups.include?("admin")
+
+        <<-EOS.undent
+          You are not a member of the "admin" group, which will cause
+          commands like `brew linkapps` to fail.
```

Comment:
That message may not make sense under `brew doctor`, but I'll add it to the `linkapps` PR as part of the output upon failure.